### ISSUE TITLE
Debug Australia distributed solar

### ISSUE THREAD

**Issue by jarek** [tmrowco/electricitymap-contrib#799]
Distributed solar production in added into Australian numbers using data from http://pv-map.apvi.org.au/live with https://github.com/tmrowco/electricitymap/blob/master/parsers/lib/AU_solar.py

However in practice it has frequent production jumps that are quite clearly missing data, for example entire solar production of Queensland disappearing in 15 minutes.

Unfortunately solar hours in Australia are middle of the night in Europe, so debugging isn't easy for me :) Can someone whose daylight overlaps with Australia's try to keep an eye on this and see if they can find what the problem is?

![queensland solar](https://user-images.githubusercontent.com/47415/32134857-1a1de6a0-bbf6-11e7-9a94-fc4709b181b2.png) ![queensland no solar](https://user-images.githubusercontent.com/47415/32134856-19fceaae-bbf6-11e7-8b4c-82f744573e54.png)

**Comment by corradio** [comment]
@jarek would it be possible to fetch values in the past to correct the last 24h?

**Comment by systemcatch** [comment]
It looks like the api returns loads of data for the whole day if we want to use it. Layout is kind of hard to understand though.

**Comment by jarek** [comment]
The solar data source does indeed return historical data, but the source we're using for rest of the generation only returns most recent datapoint. Perhaps there's a better source for Australian data than the http://services.aremi.nationalmap.gov.au/aemo/v3/csv/all CSV.

**Comment by systemcatch** [comment]
There is the api by global roam https://ausrealtimefueltype.global-roam.com/api/SeriesSnapshot?time= which provides data for http://reneweconomy.com.au/nem-watch/. We tried using it for the tesla battery but changed to another source after a while. It's 5 min aggregated by type in json format. I think you can specify historical timestamps (no luck yet).

**Comment by jarek** [comment]
So, there are currently two tickets for Australian data, the other is #1196  "AUS zones returning data from the future".

From last several weeks of casual browsing of Electricity Map I haven't seen the missing-solar issue happening. Possibly it was fixed by something along the way. Maybe it was getting confused by main data being in the future, before we started rejecting that? I'll close this issue, but please reopen if you see it happening again.

Possibly changing to different source and returning historical data is a topic more for #1196 I think. I'll post the link there.

**Comment by jarek** [comment]
I see it happening again, so reopening issue for reference, though we should probably try to fix Australia in #1196 first...

![image](https://user-images.githubusercontent.com/47415/42903722-084c9dac-8ad3-11e8-8067-062b38a9bb0e.png)

**Comment by corradio** [comment]
Is this issue still applicable with the new parser?

**Comment by KabelWlan** [comment]
I don't think this is a problem anmore with the opennem parser and therefore this issue can be closed.
